pkg/install/resources.go

```diff
@@ -105,8 +106,12 @@ func ServiceAccount(namespace string, annotations map[string]string) *corev1.Ser
 }
 
 func ClusterRoleBinding(namespace string) *rbacv1beta1.ClusterRoleBinding {
+	crbName := "velero"
+	if namespace != DefaultVeleroNamespace {
+		crbName = "velero-" + namespace
+	}
```

Comment:
```
		crbName := DefaultVeleroNamespace
		if namespace != DefaultVeleroNamespace {
			crbName = crbName + "-" + namespace
		}
```

Reply:
Hmm, this is a little awkward, because the crb and the namespace are named `velero`, but I don't think we should use the namespace constant. That, to me, creates confusion and an unclear coupling between the two objects.

For a similar example, in our standard deployment, the deployment and the namespace are both `velero`.

Reply:
I see. And that's a good point. I'll merge this.